src/main/java/io/github/cottonmc/cotton/gui/client/CottonScreen.java

```diff
@@ -16,7 +16,7 @@
 	protected WWidget lastResponder = null;
 	
 	public CottonScreen(T container, PlayerEntity player) {
-		super(container, player.inventory, new TranslatableTextComponent(""));
+		super(container, player.inventory, new StringTextComponent(""));
```

Comment:
Should there also be a constructor taking in a `TextComponent` to pass to super? That would make using `CottonScreen`s with `NameableContainerProvider` a bit easier.

Reply:
Probably not. You've already got `NameableContainerProvider` support here, the value passed up to super is just a dummy because we don't use their getter.

Reply:
Hm, yes. I currently just override `getTitle` in my screens.